distribution/docs/src/main/resources/content/_whitelists/catalog-whitelist.adoc

```diff
@@ -6,39 +6,167 @@
 :order: 01
 :summary: ${ddf-catalog} whitelist.
 
-The following packages have been exported by the ${ddf-catalog} application and are approved for use by third parties:
-
-* `ddf.camel.component.catalog`
-* `ddf.catalog`
-* `ddf.catalog.cache`
-* `ddf.catalog.data`
-* `ddf.catalog.data.metacardtype`
-* `ddf.catalog.event`
```

Comment:
❓I was looking at the documentation and we tell people to use the `ddf.catalog.event.impl.SubscriptionImpl` in the _Creating a Subscription_, so should add the class here or change the example?

Reply:
My understanding is that whitelist items should not change and since this is an implementation it can change. However that doesn't mean that it can't be used, it just needs to be used with caution. It seems odd to have an impl in the whitelist but if we need to include it I can.

Reply:
We should not have impls in the whitelist at all.